delainewendling/django_vue_tutorial
# Auth

### PR Description

This PR does a few things, it adds authentication on the backend and the frontend. I also create an axios helper class so that we do not have to think about adding the appropriate headers to every single request.

### Steps to test

1. In an incognito window go to http://localhost:8080 (or whatever port your webpack-dev-server serves up - if the port is different you will have to change your login redirect url)
1. Make sure the only options in the navbar are Home and Login
1. Click login and make sure you are redirected to the django login page
1. Once you login make sure you are redirected to the regular app and that more options are showing in the navbar
1. Go through the app and make sure there are no errors (add a new question, delete a question, edit a question)
